doc/adr/0004-marconi-initiative.rst

```diff
@@ -0,0 +1,100 @@
+=========================
+Making a case for Marconi
+=========================
+
+Plutus off-chain code oftentimes needs access to indexed portions of the blockchain. The Chain Index project is the initial solution meant to deliver access to this kind of data. However, after release, a couple of shortcomings were identified which prompted the development of an indexing solution that is based on a different set of architectural and functional constraints.
```

Comment:
```suggestion
Plutus off-chain code oftentimes needs access to indexed portions of the blockchain. The plutus-chain-index project is the initial solution meant to deliver access to this kind of data. However, after release, a couple of shortcomings were identified which prompted the development of an indexing solution that is based on a different set of architectural and functional constraints.
```

Reply:
I renamed all the `Chain Index` strings to `plutus-chain-index`. I think it looks better.